openstack/master/main.tf

```diff
@@ -49,6 +49,18 @@ resource "null_resource" "generate-inventory" {
 
 }
 
+# Upload all additional public keys to master node
```

Comment:
So the error message that I printed in the first info:

> module.master.null_resource.upload_public_keys (local-exec): cat: [path to KubeNow]/public_keys/*: No such file or directory

is what I got when not having the public_keys directory. So for me it 'failed' gracefully, it will not report this as being any error.